.github/workflows/test.yml

```diff
@@ -256,7 +256,40 @@ jobs:
         anchor deploy
       working-directory: ./integration/anchor
     - name: Create Solidity interface file for Anchor
-      run: solang idl ../anchor/target/idl/anchor.json
+      run: solang idl ../../anchor/target/idl/anchor.json
+      working-directory: ./integration/solana/anchor
+    - name: Build Solang contracts
+      run: solang compile --target solana *.sol -v -o ../
+      working-directory: ./integration/solana/anchor
+    - name: Deploy and test contracts
+      run: npm run test-anchor
+      working-directory: ./integration/solana
```

Comment:
Does this test really work, there is no test validator running. Also the test seem broken (I don't now why)

Reply:
Yes, the test works. Please, see the CI output. The validator is run with the command `nohup solana-test-validator -q &`.

Reply:
For some reason, the tests are not shown in the PR, but they ran here:
https://github.com/hyperledger/solang/actions/runs/3189875558/jobs/5204512205

Reply:
Everything ran fine now.